pkg/network/http/monitor_test.go

```diff
@@ -276,40 +277,79 @@ func TestUnknownMethodRegression(t *testing.T) {
 	// SetupDNAT sets up a NAT translation from 2.2.2.2 to 1.1.1.1
 	netlink.SetupDNAT(t)
 
-	targetAddr := "2.2.2.2:8080"
-	serverAddr := "1.1.1.1:8080"
+	targetAddr := "2.2.2.2:8888"
+	serverAddr := "1.1.1.1:8888"
 	srvDoneFn := testutil.HTTPServer(t, serverAddr, testutil.Options{
 		EnableTLS:        false,
 		EnableKeepAlives: true,
 	})
 	defer srvDoneFn()
 
+	/* Save and recover TCP timestamp option */
+	oldTCPTS, err := exec.Command("cat", "/proc/sys/net/ipv4/tcp_timestamps").CombinedOutput()
```

Comment:
it's great that we're testing this, but could you place this into a separate test?

Reply:
I will move this in the testutil.HTTPServer(Options{EnableTCPTimestamp: true}), so we can modulate the HTTPServer tests